src/components/AboutPage.js

```diff
@@ -1,15 +1,136 @@
 import React from 'react';
+import { Button, Col, Container, Row } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+import { Link } from 'react-router-dom';
+{/* TODO
+  - images
+  - Contact Us icons
+  - style cleanup
+*/}
 
 const AboutPage = () => (
-  <div class="container mt-5">
-    <div class="row">
-      <div class="col"></div>
-      <div class="col-8">
-        <h1>About us</h1>
-      </div>
-      <div class="col"></div>
-    </div>
-  </div>
+  <Container className="container" fluid="md">
+
+    <Row>
+      {/* TODO place image here (logo) */}
+      <Col className="col"></Col>
+      <Col className="col-8">
+        <h1>Build for Black Lives</h1>
+        <p>
+          Build for Black Lives, a platform created by HackBeanpot, connects 
+          individuals who have tech/design skills with activists, organizers, 
+          and Black-owned businesses. We aim to provide pro bono work creating 
+          websites, mobile apps, data visualizations, graphics, digital 
+          marketing, and/or any other technical projects the organizations might
+           need. The length of these projects could range from a few days to 
+           more long term.
+          </p>
+          <p>
+          HackBeanpot is a Boston-based hackathon, and as an organization we aim
+           to build an inclusive community and provide resources that allow our 
+          community to support each other in challenging times. Racial 
+          inequality has long been an issue in the tech space. While change 
+          does not happen overnight, we want to make an active effort to bring 
+          the tech community together to help support Black lives and fight for 
+          racial justice.
+        </p>
+      </Col>
+    </Row>
+
+    <hr />
+    
+    <Row>
+      <Col className="col"></Col>
+      <Col className="col-10">
+        <h2>Community Partners</h2>
+        <p>
+          These are organizations that have helped build, endorse, and promote 
+          the platoform.
+        </p>
+        <ul>
+          <li>HackBeanpot</li>
```

Comment:
this is nitpicky but since the background is black you can't see the bullet so it just looks a little awkward spaced inline w/ only one partner - could we maybe add something like "contact us if you want to get involved or join our slack" kind of language for people who want to be community partners? And then unindent HBP for now or change the bullet color?

Reply:
O I removed the bullets, but I'll un-indent to better match the wireframe.
When we ask them to reach out, should we link our email again, or would that be redundant since it's already mentioned later on the same page?